lago/prefix.py

```diff
@@ -311,9 +311,17 @@ def _add_nic_to_mapping(self, net, dom, nic):
 
         Returns:
             None
+
+        Raises:
+            RuntimeError: If the provided domain name is empty string
         """
         dom_name = dom['name']
         idx = dom['nics'].index(nic)
+        if not dom_name:
+            raise RuntimeError(
+                    'Invalid domain name. A name must be specified for the domain'
```

Comment:
The message follows the format ```virt-manager``` is using when creation of guest without name is attempted. Just replaced (s/guest/domain/) in that message for purpose of lago.